console/frontend/src/main/frontend/src/app/views/jms/jms-send-message/jms-send-message.component.ts

```diff
@@ -90,18 +92,22 @@ export class JmsSendMessageComponent implements OnInit {
       return;
     }
 
-    this.jmsService.postJmsMessage(fd).subscribe({ next: (returnData) => {
-      this.error = "";
-      this.processing = false;
-    }, error: (errorData: HttpErrorResponse) => {
-      this.processing = false;
-      const error = errorData.error ? errorData.error.error : "An unknown error occured, check the logs for more info.";
-      this.error = typeof error === 'object' ? error.error : error;
-    }});
+    this.jmsService.postJmsMessage(fd).subscribe({
+      next: (returnData) => {
+        this.error = "";
+        this.processing = false;
+        this.successMessage = 'JMS Message sent successfully';
+      }, error: (errorData: HttpErrorResponse) => {
+        this.processing = false;
+        const error = errorData.error ? errorData.error.error : "An unknown error occured, check the logs for more info.";
```

Comment:
Didn't the linter pick up the quotes?

Reply:
Oh wait, we didn't backport that, right?

Reply:
Nope we didn't, sadly it looks a bit messy but hopefully not important anymore soon. I hope that with the 8.1 release we dont need to think about backporting to 8.0 or 7.9 anymore